api/grpc/grpc.go

```diff
@@ -13,19 +13,33 @@ import (
 	grpcprometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	log "github.com/sirupsen/logrus"
+	"github.com/stackrox/scanner/pkg/env"
 	"github.com/stackrox/scanner/pkg/mtls"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 )
 
 const (
-	localEndpoint = "127.0.0.1:8444"
+	localEndpoint                   = "127.0.0.1:8444"
+	defaultGrpcMaxConcurrentStreams = 100 // HTTP/2 spec recommendation for minimum value
 )
 
 func init() {
 	grpcprometheus.EnableHandlingTimeHistogram()
 }
 
+var (
+	maxGrpcConcurrentStreamsSetting = env.RegisterIntegerSetting("ROX_GRPC_MAX_CONCURRENT_STREAMS", defaultGrpcMaxConcurrentStreams)
+)
+
+func maxGrpcConcurrentStreams() uint32 {
+	if maxGrpcConcurrentStreamsSetting.Int() < 0 {
```

Comment:
Though the gRPC library defaults this to `math.MaxUint` when 0, the HTTP/2 library defaults it to at least 100 when 0. Can we also default it to 100?

Reply:
Are you suggesting that I change `< 0` to`<= 0`?

Reply:
yep

Reply:
Yeah I missed that. I should do it also in stackrox.